app/controllers/idv/personal_key_controller.rb

```diff
@@ -1,11 +1,9 @@
 module Idv
   class PersonalKeyController < ApplicationController
-    include IdvSession
+    include IdvStepConcern
```

Comment:
`IdvStepConcern` adds some `before_action`s which restrict a user who already has a profile, and since the profile would be activated by this point, they're being redirected (hence the failing build). 

Alternatives to consider:

- Create a separate concern for `step_indicator_steps`
- Consider loosening restrictions in `IdvStepConcern` or finding another way to allow it to be included here
- Duplicate method in this controller

Reply:
I addressed this by creating a separate `Idv::StepIndicatorConcern` to handle step indicator logic.